packages/office-ui-fabric-react/src/components/SwatchColorPicker/ColorPickerGridCell.base.tsx

```diff
@@ -101,6 +102,16 @@ export class ColorPickerGridCellBase extends React.Component<IColorPickerGridCel
     return inputColor!.toLocaleLowerCase() === '#ffffff';
   }
 
+  /**
+* Validate if the cell's color is dark enough to apply the dark cell high contrast style
+* @param inputColor - The color of the current cell
+* @returns - Whether the cell's color is dark or not.
+*/
+  private _isDarkCell(inputColor: string | undefined): boolean {
+    let color = inputColor ? getColorFromString(inputColor) : undefined;
+    return (color && (color.r < 120 && color.g < 120 && color.b < 120)) || false;
```

Comment:
Let's run this by designers to make sure everyone is on the same page around what constitutes "dark".  FYI, @betrue-final-final

Reply:
@betrue-final-final we need to make a call on when we are going to add a white border to the dark circle(s)... Should it be only for the black circle or if the color is darker than a specific hue? ... and if we go the hue route, should we update the isWhite logic to be isLight?

Reply:
I'm going to explore looking at the white border. We should not change swatches to be more visible but remove any accuracy.